qa-pipelines/tasks/delete-aks-resource-group.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Why can't we just use terraform for teardown?

Reply:
Cause that will require us to use public cloud to save the state of terraform. Which we don't want to do. An architectural decision was made to not use such resource as its an over kill for our needs. 
In addition, terrform  teardown might leave leftover resources as seen in past whereas use az to delete resource group is very effective.